public/components/datasources/components/manage/accelerations/utils/acceleration_utils.tsx

```diff
@@ -93,6 +93,7 @@ export const CreateAccelerationFlyoutButton = ({
   dataSourceName: string;
   renderCreateAccelerationFlyout: (
     dataSource: string,
+    dataSourceClientId?:string,
```

Comment:
are these two data source the same concept? or i think the first one is for flint and second one is MDS?

if that's the case maybe add MDS in the variable name or add a comment somewhere